types/time.js

```diff
@@ -138,21 +143,24 @@ class TimeType extends Commando.ArgumentType {
 	}
 
 	static generateTimes(possible_date) {
-		const possible_dates = [possible_date],
-			ambiguously_am = possible_date.hour() < 12 &&
-				!possible_date.creationData().format.endsWith('a');
+		const possible_dates = [],
+			date_format = possible_date.creationData().format,
+			hour = possible_date.hour(),
+			ambiguously_am = hour < 12 && !date_format.endsWith('a');
```

Comment:
Just curious, but why would the formatted date end with the letter "a"? is it like 10:20a instead of AM or PM?

Reply:
It won't necessarily - that's moment-style formatting that it's looking at (https://momentjs.com/docs/#/parsing/ - scroll down a bit more to find it).  That just means if moment detected the parsed time format as one that explicitly has am/pm in it then even if the time is before noon it's not ambiguous - it's explicitly an AM time.